openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -224,11 +226,24 @@ def _track_message_sent(site, user, msg):
         properties['course_ids'] = course_ids[:10]
         properties['primary_course_id'] = course_ids[0]
 
-    analytics.track(
-        user_id=user.id,
-        event='edx.bi.email.sent',
-        properties=properties
-    )
+    tracking_context = {
+        'host': site.domain,
```

Comment:
Will this always match the value in the other events?

Reply:
I don't know how to address "always", but it seems closest to the "truth" in the case of these emails.   At the moment, these events don't provide the information, so it's hard to evaluate empirically.   But going forward we'll be able to assess that.